src/pages/resources/todo-app/lesson2.md

```diff
@@ -40,7 +40,7 @@ npm i --save @adobe/aio-lib-state
 
 Then we're going to import it as well:
 
-``javascript
+```javascript
 const stateLib = require('@adobe/aio-lib-state');
 ``
```

Comment:
thanks! can you update line 45 as well to have three backticks to close the block?

Reply:
missed that, added now.